UI/src/view/municipalityWorkList.js

```diff
@@ -115,6 +118,10 @@
         });
       };
 
+      var suggestedAssetsButton = function(counter, typeId) {
```

Comment:
This suggestedAssets button needs to be removed because the user story who have this functionality will not be deployed first then this user story.